usaspending_api/search/delta_models/award_search.py

```diff
@@ -1,126 +1,131 @@
-    "pop_state_code": {"delta": "STRING", "postgres": "TEXT"},
-    "pop_county_code": {"delta": "STRING", "postgres": "TEXT"},
-    "pop_county_name": {"delta": "STRING", "postgres": "TEXT"},
-    "pop_city_code": {"delta": "STRING", "postgres": "TEXT"},
-    "pop_zip5": {"delta": "STRING", "postgres": "TEXT"},
-    "pop_congressional_code": {"delta": "STRING", "postgres": "TEXT"},
-    "pop_city_name": {"delta": "STRING", "postgres": "TEXT"},
-    "pop_state_name": {"delta": "STRING", "postgres": "TEXT"},
-    "pop_state_fips": {"delta": "STRING", "postgres": "TEXT"},
-    "pop_state_population": {"delta": "INTEGER", "postgres": "INTEGER"},
-    "pop_county_population": {"delta": "INTEGER", "postgres": "INTEGER"},
-    "pop_congressional_population": {"delta": "INTEGER", "postgres": "INTEGER"},
-    "cfda_program_title": {"delta": "STRING", "postgres": "TEXT"},
-    "cfda_number": {"delta": "STRING", "postgres": "TEXT"},
-    "cfdas": {"delta": "ARRAY<STRING>", "postgres": "TEXT[]"},
-    "sai_number": {"delta": "STRING", "postgres": "TEXT"},
-    "type_of_contract_pricing": {"delta": "STRING", "postgres": "TEXT"},
-    "extent_competed": {"delta": "STRING", "postgres": "TEXT"},
-    "type_set_aside": {"delta": "STRING", "postgres": "TEXT"},
-    "product_or_service_code": {"delta": "STRING", "postgres": "TEXT"},
-    "product_or_service_description": {"delta": "STRING", "postgres": "TEXT"},
-    "naics_code": {"delta": "STRING", "postgres": "TEXT"},
-    "naics_description": {"delta": "STRING", "postgres": "TEXT"},
-    "tas_paths": {"delta": "ARRAY<STRING>", "postgres": "TEXT[]"},
-    "tas_components": {"delta": "ARRAY<STRING>", "postgres": "TEXT[]"},
-    "disaster_emergency_fund_codes": {"delta": "ARRAY<STRING>", "postgres": "TEXT[]"},
-    "covid_spending_by_defc": {"delta": "STRING", "postgres": "JSONB"},
-    "total_covid_outlay": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)"},
-    "total_covid_obligation": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)"},
-    "officer_1_amount": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)"},
-    "officer_1_name": {"delta": "STRING", "postgres": "TEXT"},
-    "officer_2_amount": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)"},
-    "officer_2_name": {"delta": "STRING", "postgres": "TEXT"},
-    "officer_3_amount": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)"},
-    "officer_3_name": {"delta": "STRING", "postgres": "TEXT"},
-    "officer_4_amount": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)"},
-    "officer_4_name": {"delta": "STRING", "postgres": "TEXT"},
-    "officer_5_amount": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)"},
-    "officer_5_name": {"delta": "STRING", "postgres": "TEXT"},
+    "treasury_account_identifiers": {"delta": "ARRAY<INTEGER>", "postgres": "INTEGER[]", "gold": False},
+    "award_id": {"delta": "LONG NOT NULL", "postgres": "BIGINT NOT NULL", "gold": True},
```

Comment:
`award_id` was on the original table, so `gold` should be `False` here right?

Reply:
Same with a few fields below such as `category`, `type`, `type_description`, etc